prime-angular-frontend/src/app/modules/enrolment/shared/components/obo-site-form/obo-site-form.component.html

```diff
@@ -24,14 +24,33 @@
 
           <mat-form-field class="w-100">
             <input matInput
-                   placeholder="Health Authority Facility Name"
+                   placeholder="'{{healthAuthorityName}}' Health Authority Facility Name"
                    formControlName="facilityName">
             <mat-error>Required</mat-error>
           </mat-form-field>
 
         </div>
       </div>
 
+      <mat-form-field class="w-100">
```

Comment:
Needs a bit of vertical spacing otherwise it's sitting right on top of the next field:

![image](https://user-images.githubusercontent.com/599498/116457673-8d67be80-a818-11eb-94a0-a537ee4640b8.png)

The spacing should match the kind of spacing seen when the error message appears so the page doesn't move when the validation message appears:

![image](https://user-images.githubusercontent.com/599498/116457768-ab352380-a818-11eb-91eb-593154f6b71c.png)

Reply:
Addressed I believe